README.md

```diff
@@ -213,6 +247,25 @@ swift run --product AppKid
 There are additional scripts under `Utilities` directory with pre-defined building, running, cleaning and other commands.
 > **WARNING:** Building on macOS is broken at the moment because of the issue of using vulkan-sdk as C library in pure swift project
 
+### TroubleShooting
+Having trouble with your swift build?   
+Seems to be pulling an older version of AppKid from a cache somewhere?   
+Want to make sure you are on the latest from git ?     
+Updating from an older version?   
+
+You might need to re-build yourapp, [eg MyApp] , with :
+
+```bash
+    swift package update 
+    swift package purge-cache 
+    swift package reset 
+    swift package clean 
+    rm -rf ~/.cache/org.swift.foundation.URLCache/
+    rm -rf ~/.cache/org.swift.swiftpm/
+    rm -rf .build/
```

Comment:
You want to update packages only after you've purged your caches. Otherwise update you've performed is literally gone